docs/source/en/using-diffusers/kerascv.mdx

```diff
@@ -0,0 +1,171 @@
+More specifically, it first converts the checkpoints to PyTorch and then wraps them into a
+[`StableDiffusionPipeline`](https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/overview) which is ready
+for inference. Finally, it pushes the converted checkpoints to a repository on the Hugging Face Hub. 
+
+We welcome you to try out the tool [here](https://huggingface.co/spaces/sayakpaul/convert-kerascv-sd-diffusers)
+and share feedback via [discussions](https://huggingface.co/spaces/sayakpaul/convert-kerascv-sd-diffusers/discussions/new). 
+
+## Getting started 
+
+First, you need to obtain the fine-tuned KerasCV Stable Diffusion checkpoints. We provide an
+overview of the different ways Stable Diffusion models can be fine-tuned [here](https://huggingface.co/docs/diffusers/training/overview). For Keras implementations of some of these methods, you can check out the resources below:
+
+* [Teach StableDiffusion new concepts via Textual Inversion](https://keras.io/examples/generative/fine_tune_via_textual_inversion/)
+* [Fine-tuning Stable Diffusion](https://keras.io/examples/generative/finetune_stable_diffusion/)
+* [DreamBooth](https://keras.io/examples/generative/dreambooth/)
+* [Prompt-to-Prompt editing](https://github.com/miguelCalado/prompt-to-prompt-tensorflow)
+
+A Stable Diffusion model has three primary components:
+
+* Text encoder 
+* UNet 
+* VAE 
+
+Depending on the fine-tuning task, we may fine-tune one of more of these components (the VAE is almost always left untouched). Here are some common combinations:
+
+* DreamBooth: Text encoder and UNet 
+* Classical text2image: UNet 
+* Textual Inversion: Text encoder 
+
+Let's use [this checkpoint](https://huggingface.co/sayakpaul/textual-inversion-kerasio/resolve/main/textual_inversion_kerasio.h5) which was generated
+by conducting Textual Inversion with the following "placeholder token": `<my-funny-cat-token>`. 
+
+On the tool, we supply the following things: 
+
+* Path(s) to download the fine-tuned checkpoint(s) (KerasCV)
+* An HF token 
+* Placeholder token (only applicable for Textual Inversion)
+
+<div align="center">
+<img src="https://i.imgur.com/JMATVKJ.png"/>
+</div> 
+
+As soon as you hit "Submit", the conversion process will begin. Once it's complete, you should see the following:
+
+<div align="center">
+<img src="https://i.imgur.com/gJyVD9g.png"/>
+</div> 
+
+If you click the [link](https://huggingface.co/sayakpaul/textual-inversion-cat-kerascv_sd_diffusers_pipeline/tree/main), you
+should see something like so: 
+
+<div align="center">
+<img src="https://i.imgur.com/vl6w7Zm.png"/>
+</div> 
+
+If you head over to the [model card of the repository](https://huggingface.co/sayakpaul/textual-inversion-cat-kerascv_sd_diffusers_pipeline), the
+following should appear: 
+
+<div align="center">
+<img src="https://i.imgur.com/PxCNRFh.png"/>
+</div> 
+
+_**Note that we're not specifying the UNet weights here since the UNet is not fine-tuned during Textual Inversion.**_
```

Comment:
I don't understand what this means, sorry. Is this about the first step in the conversion process? If so, I'd put this comment (maybe as a `tip`) below the corresponding screenshot.